src/app/jumio/jumio-error/jumio-error.component.html

```diff
@@ -0,0 +1,17 @@
+<app-banner></app-banner>
+<div class="container home-container">
+  <div class="fs-24px font-weight-bold pt-15px">Jumio Verification Failed</div>
+  <div class="d-flex align-items-center">
+    <div class="fs-15px mt-15px">
+      We were unable to verify your identity. You can click close to proceed to {{ hostname }}
```

Comment:
@diamondhands0 Is this the copy we want?

Reply:
I wasn't able to trigger this path. Where does it even happen?

Reply:
it happens when you fail to get through the jumio flow.